locales/en/plugin__kubevirt-plugin.json

```diff
@@ -864,6 +864,7 @@
   "Requested of ": "Requested of ",
   "Required": "Required",
   "Required when automatic update is enabled": "Required when automatic update is enabled",
+  "Requires RHEL9 and QEMU guest agent. If enabled, SSH keys can be changed while the VirtualMachine is running. Otherwise, the VirtualMachine inherits the key injection setting of its image.": "Requires RHEL9 and QEMU guest agent. If enabled, SSH keys can be changed while the VirtualMachine is running. Otherwise, the VirtualMachine inherits the key injection setting of its image.",
```

Comment:
I can see "RHEL9" is used here as a part of the popover text, but another popover text uses "RHEL 9" (you can see that text a few lines above, line 758) which looks inconsistent. Could you please, find out what is the correct form (maybe asking Avital from doc team or Yifat someone from UX) and update the code? Thanks! :)

Reply:
"RHEL 9", not "RHEL9"